includes/Plugin.php

```diff
@@ -216,6 +216,9 @@ function ( $args ) {
 		if ( defined( 'WP_CLI' ) && WP_CLI ) {
 			( new \Google\Site_Kit\Core\CLI\CLI_Commands( $this->context ) )->register();
 		}
+
+		// Add Plugin Row Meta.
+		( new \Google\Site_Kit\Core\Admin\PluginRowMeta() )->register();
```

Comment:
The `\Google\Site_Kit\` prefix of the class name can be removed since this class is already in this namespace, a relative class name can be used like with `Activation_Flag` and `Uninstallation` above. Would you please do the same for `CLI_Commands` as well?